lib/rubocop/directive_comment.rb

```diff
@@ -54,6 +55,11 @@ def disabled?
       %w[disable todo].include?(mode)
     end
 
+    # Checks if this directive disables cops only on next line
+    def disabled_next_line?
+      %w[disable-next-line].include?(mode)
+    end
```

Comment:
I think we should support `todo-next-line` as well to mirror how `disable`/`todo` work.

Reply:
Supported